bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/AndroidTVHandler.java

```diff
@@ -157,14 +157,17 @@ public void initialize() {
 
         GoogleTVConfiguration googletvConfig = getConfigAs(GoogleTVConfiguration.class);
         String ipAddress = googletvConfig.ipAddress;
+        boolean gtvEnabled = (googletvConfig.gtvEnabled) ? true : false;
```

Comment:
I think this can be simplified :slightly_smiling_face: 

```suggestion
        boolean gtvEnabled = googletvConfig.gtvEnabled
```

Reply:
It was in there strictly as a safety check against a null value.

Reply:
It can never be `null` because the field is a primitive in `GoogleTVConfiguration` (it's `boolean` and not `Boolean`).

```java
    public boolean gtvEnabled;
```

Reply:
That's true. I used the same check elsewhere so just kept going without thinking about that.

Reply:
Even if it was a `Boolean` and the value was `null`, Java will throw a `NullPointerException` when unboxing it, because `null` cannot be evaluated to `true` or `false` (unlike how this works in other languages):

![image](https://github.com/openhab/openhab-addons/assets/12213581/9b85582e-2661-49d1-b383-1458db8b6129)